docs/technical_reference/property_models/modified_ADM1.rst

```diff
@@ -0,0 +1,307 @@
+Modified ADM1 Property Package
+==============================
+
+This package is an extension of the base ADM1 model and implements properties and reactions of an anaerobic digestion model for wastewater treatment using an anaerobic digester as provided in
+`Batstone, D. J. et al. (2002) <https://iwaponline.com/wst/article-abstract/45/10/65/6034>`_ and `Rosen and Jeppsson (2006) <https://www.iea.lth.se/WWTmodels_download/TR_ADM1.pdf>`_.
+
+This Anaerobic Digestion Model no.1 (ADM1) makes the following modifications to the base ADM1 model as provided in: `Flores-Alsina, X. et al. (2016) <https://www.sciencedirect.com/science/article/pii/S0043135416301397>`_
+   * tracks inorganic phosphorus (S_IP), polyhydroxyalkanoates (X_PHA), polyphosphates (X_PP), phosphorus accumulating organisms (X_PAO), potassium (S_K), and magnesium (S_Mg)
+   * removes the composite material variable (X_C) and the associated disintegration reaction
+   * adds 7 additional reactions
```

Comment:
Another thought I had-- which is just a suggestion--  I thought it could be useful to list all of the components/reactions that are the same as in the original ADM1 model, then highlight the new additions (or deletions) before or after a given table listing model content. Does that make sense? I think it could be useful for quickly determining the differences.

Reply:
That makes sense.

Reply:
@adam-a-a I added the color formatting rather than additional tables. Hopefully you agree that it's more visually appealing while still being able to clearly see the modifications made to the base model? Are the colors themselves fine though?

Reply:
This is great. One thing I will note is that the green is hard to distinguish from the nominal black text (at least on my screen), but this is a big improvement over what we had earlier. 

Another thing that I think would be useful is to insert a reminder note under each section as to what the colors are. For example, if a user is skimming through the sections, it would be convenient if the color reminder was in the relevant section where those colors are used, in case they skipped past the top-of-page description. For example, you can add such reminders at the beginning of a section, only for the color text that is included in that section.